pkg/apiserver/api/v1/account.proto

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2019 The KubeCarrier Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+package kubecarrier.api.v1;
+option go_package = "v1";
+
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
+import "meta.proto";
+import "types.proto";
+import "request.proto";
+
+message Account {
+  ObjectMeta metadata = 1;
+  AccountSpec spec = 2;
+  AccountStatus status = 3;
+}
+
+message AccountSpec {
+  AccountMetadata metadata = 1;
+  repeated AccountRole roles = 2;
+  repeated Subject subjects = 3;
+}
+
+message AccountRole{
+  string type = 1;
+}
+
+message AccountMetadata {
+  string displayName = 1;
+  string description = 2;
+  string shortDescription = 3;
+  Image logo = 4;
+  Image icon = 5;
+}
+
+message Subject {
+  string kind = 1;
+  string apiGroup = 2;
+  string name = 3;
+  string namespace = 4;
+}
+
+message AccountStatus {
+  string phase = 1;
+  repeated AccountCondition conditions = 2;
+}
+
+message AccountCondition {
+  AccountConditionType type = 1;
+  ConditionStatus status = 2;
+  google.protobuf.Timestamp lastTransitionTime = 3;
+  string reason = 4;
+  string message = 5;
+}
+
+message ConditionStatus {
```

Comment:
This could be moved to `types.proto` or something like that; no need to repeat it everywhere.

Reply:
fixed